includes/definitions/sophos-xg.yaml

```diff
@@ -0,0 +1,13 @@
+os: sophos-xg
+text: 'Sophos XG'
+type: appliance
+icon: sophos-xg
+group: sophos
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.2604.5
+mib_dir: sophos-xg
```

Comment:
you should create this dir and put the mib in it

Reply:
@murrant I didn't found how to create a folder under mibs, could you help me, or do your own and move MIB file into it?
Thanks

Reply:
You create a folder under mibs and put it there

Reply:
also, mib_dir is not needed, if it matches the os name. BUT, this mib should just be in the sophos folder.

Reply:
Understood.
Done.

Reply:
Directory renamed to **sophos**. 
Still present mib_dir because os name do not match directory name.